openapi-v1.yaml

```diff
@@ -6424,48 +6820,95 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
-  /groups/{namespace}/{name}/{asset_namespace}/{asset_name}:
+  /groups/{group_namespace}/{group_name}/arrays/{namespace}/{array}:
```

Comment:
Why do we need separate post routes for arrays and groups?

Reply:
Two reason
1. middleware. ArrayEnforcer expects `namespace,name`
2. backend would know which table to search. Otherwise it must search group, arrays and resolve any conflict in case the user passes a name